### DISCUSSION TITLE
Poll State Node - Update interval from entity

### DISCUSSION THREAD

**Discussion by aperture1988** [zachowj/node-red-contrib-home-assistant-websocket#370] [Feature Request]
Hey Guys,

Would be brilliant if the Poll State node Update Interval value could be set using a Home Assistant entity. Similar to the way the time node operates.

For Example:

I have a card set up to control the the Air Freshener automation in Node Red from my Home Assistant front end. I am using a slider to set the frequency I want them to trigger. If the Update Interval under the Poll State node allowed me to reference the value of this slider the entire automation could be controlled from home assistant. Currently using an inject node to set an interval but can only change this value from within the Node Red platform.

![image](https://user-images.githubusercontent.com/51124619/114097997-ac79cd00-98b8-11eb-8de0-edd21c834c89.png)

**Comment by zachowj** [comment]
Added in v0.38.0
